README.adoc

```diff
@@ -82,3 +82,17 @@ You can now open the URL from HOST/PORT in your browser:
 http://registration-service-toolchain-host-operator.192.168.42.237.nip.io
 
 _Note_: the actual route/URL will be different in your minishift.
+
+=== Deploying all operators and Registration Service
+
+All e2e resources sush as Host operator, Member operator, Registration Service, CRDs, etc can be deployed via:
```

Comment:
All e2e resources sush as Host operator, Member operator, Registration Service, CRDs, etc can be deployed via:
```suggestion
All e2e resources such as Host operator, Member operator, Registration Service, CRDs, etc can be deployed via:
```